src/components/Headline/Headline.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+Headline.propTypes = {
+  childre: PropTypes.node.isRequired,
+  small: PropTypes.bool,
+}
+
+export default function Headline({ children, small }) {
+  return (
+    <Header>
+      <Text small={small}>{children}</Text>
```

Comment:
Are you sure you need the header element around the h1? 
I think only a h1 or a header element would be enough.

Reply:
Very good observation. That h1 was unnecessary and i removed it. Thank you!